platform/android/MapLibreAndroidTestApp/src/main/java/org/maplibre/android/testapp/activity/style/SymbolGeneratorActivity.kt

```diff
@@ -139,7 +139,7 @@ class SymbolGeneratorActivity : AppCompatActivity(), OnMapReadyCallback {
      * Utility class to generate Bitmaps for Symbol.
      *
      *
-     * Bitmaps can be added to the map with [com.mapbox.mapboxsdk.maps.MapboxMap.addImage]
+     * Bitmaps can be added to the map with [MapLibreMap]
```

Comment:
Was this change fully intentional?

Reply:
Yes. It works this way.

Reply:
Oh I see. This is not the same.

Reply:
I don't think this method ever existed. 😛